include/grpc/event_engine/event_engine.h

```diff
@@ -166,14 +166,17 @@ class EventEngine {
     /// been called.
     virtual absl::Status Bind(const ResolvedAddress& addr) = 0;
     virtual absl::Status Start() = 0;
-    virtual absl::Status Shutdown() = 0;
   };
 
   // TODO(hork): define status codes for the callback
   // TODO(hork): define return status codes
   // TODO(hork): document status arg meanings for on_accept and on_shutdown
   /// Factory method to create a network listener.
-  virtual absl::StatusOr<Listener> CreateListener(
+  ///
+  /// The EventEngine implementation is responsible for defining the lifteime of
```

Comment:
I don't think this comment is needed.  The fact that the return type is a `unique_ptr<>` is basically self-documenting, and enforced by the compiler.

Reply:
Done